src/test/java/com/github/mauricioaniche/ck/RFCTest.java

```diff
@@ -16,49 +15,70 @@ public static void setUp() {
 	}
 
 	@Test
+	@DisplayName("Counts the number of methods invocations")
 	public void countMethodInvocations() {
 		CKClassResult a = report.get("rfc.GO");
 		Assertions.assertEquals(3, a.getRfc());
 	}
 
 	@Test
+	@DisplayName("Counts the number of methods invocations at method level")
+	public void methodLevel() {
+		CKClassResult a = report.get("rfc.GO");
+		Assertions.assertEquals(2, a.getMethod("m1/0").get().getRfc());
+		Assertions.assertEquals(0, a.getMethod("m2/1[int]").get().getRfc());
+		Assertions.assertEquals(1, a.getMethod("m3/0").get().getRfc());
+	}
+
+	@Test
+	@DisplayName("Counts the number of super methods invocations")
 	public void countSuperInvocations() {
 		CKClassResult a = report.get("rfc.GO3");
 		Assertions.assertEquals(2, a.getRfc());
 	}
 
 	@Test
```

Comment:
Oh, if it's returning 2, it's because we are now able to differentiate it. Maybe the new method name function made the trick. 

This is an exceptional test case, as `a()` is not available, so JDT has to guess things. Suggestion: make the assertion to 2 again, and just rename it to `methodCallsToAMethodNotInTheProject`, or something like that.